src/Workspaces/SharedUtilitiesAndExtensions/Workspace/Core/CodeGeneration/AbstractCodeGenerationService.cs

```diff
@@ -248,11 +256,13 @@ protected static T Cast<T>(object value)
 
             var newDocument = oldDocument.WithSyntaxRoot(currentRoot);
 
+#if !CODE_STYLE
             if (context.Context.AddImports)
             {
                 var addImportsOptions = await newDocument.GetAddImportPlacementOptionsAsync(context.FallbackOptions, cancellationToken).ConfigureAwait(false);
                 newDocument = await ImportAdder.AddImportsFromSymbolAnnotationAsync(newDocument, addImportsOptions, cancellationToken).ConfigureAwait(false);
             }
+#endif
```

Comment:
carved out as it's not needed by the feature we actually moved down, and because this led to pulling in a huge amount of additional code.

Reply:
Do we want a tracking issue to make this available in code style layer with a follow-up change? Might lead to subtle bugs if we move down a fixer to the shared layer that relies on this codegen option.

Reply:
Sure. I'll file follow-up!